internal/brokers/broker_test.go

```diff
@@ -0,0 +1,300 @@
+package brokers_test
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd/internal/brokers"
+	"github.com/ubuntu/authd/internal/responses"
+	"github.com/ubuntu/authd/internal/testutils"
+)
+
+func TestNewBroker(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string]struct {
+		name       string
+		configFile string
+		wantErr    bool
+	}{
+		"No config means local broker":                        {name: "local"},
+		"Successfully create broker with correct config file": {name: "broker", configFile: "Broker"},
+
+		"Error when config file is invalid": {configFile: "invalid", wantErr: true},
```

Comment:
config file not existant too? (even if the manager should protect against that, we are clear on our Broker object behaviour, that way)